src/v2/view-builder/views/idp/AuthenticatorIdPView.js

```diff
@@ -18,6 +18,29 @@ const Body = BaseForm.extend({
       : loc('oie.idp.enroll.description', 'login', [displayName]);
   },
 
+  showMessages () {
+    // IdP Authenticator error messages are not form errors
+    // Parse and display them here.
+    const messages = this.options.appState.get('messages') || {};
+    if (Array.isArray(messages.value)) {
+      this.add('<div class="ion-messages-container”></div>', '.o-form-error-container');
+
+      messages
```

Comment:
Do you think it would be better to enhance `BaseForm` `showMessages` function to cover this case ??

Reply:
Yes! That would be ideal. However, I need to understand the history of the `BaseForm` implementation and why we and UX signed off on errors being surfaced in that way. Adding this as a fast-follow action item for Widget cleanup!

Reply:
Much appreaited!!

Reply:
IIRC the showMessages in the BaseForm by default will render messages as a text. This is used in a few scenarios example password expiry message which can be reproduced using `authenticator-expiry-warning-password.json`

<img width="445" alt="Screen Shot 2021-03-11 at 1 01 18 PM" src="https://user-images.githubusercontent.com/23267876/110854542-f5af1080-8269-11eb-8e2b-3009c40fb406.png">

In this case the "class":"INFO". When we clean Baseform to consolidate all the error logic, up we can make use of this class attribute to render text if its `INFO` and render the error callout if class is `ERROR`